### ISSUE TITLE
Wrong curly bracket color

### ISSUE THREAD

**Issue by umpirsky** [atom/one-dark-syntax#101]
### Description

Open `{` is yellow for class, but gray for methods.

![screenshot from 2017-09-20 18-57-42](https://user-images.githubusercontent.com/208957/30683586-45bc4168-9eae-11e7-83da-526a1342dcee.png)

### Steps to Reproduce

This started happening when I upgraded to `1.21`.

1. Open any PHP file (e.g. https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php)

**Expected behavior:** Both brackets to be of same color.

**Actual behavior:** See description.

**Reproduces how often:** Always.

See https://github.com/atom/atom/issues/15624 for more details.

### Versions

```
atom-beta --version
Atom    : 1.21.0-beta1
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
```
```
apm --version
apm  1.18.4
npm  3.10.10
node 6.9.5 x64
python 2.7.12
git 2.7.4
```
```
cat /etc/issue
Ubuntu 16.04.3 LTS \n \l
```

**Comment by rafaelrenanpacheco** [comment]
Brackets are still yellow in Atom 1.21.1

**Comment by umpirsky** [comment]
This is the default syntax for Atom editor, right?

**Comment by tafelglotzer** [comment]
Still yellow in 1.22.1

**Comment by 50Wliu** [comment]
> This is the default syntax for Atom editor, right?

I believe so, yes.

**Comment by simurai** [comment]
Should be fixed part of Atom `1.24`. In the meantime you can add the following to your [styles.less](http://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks) file:

```less
// Fix curly brackets in classes (PHP)
.syntax--class.syntax--bracket.syntax--php {
  color: #abb2bf;
}
```
